packages/mui-core/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -945,8 +945,19 @@ export default function useAutocomplete(props) {
   };
 
   const handleInputMouseDown = (event) => {
-    if (inputValue === '' || !open) {
-      handlePopupIndicator(event);
+    const className = event.target.className;
```

Comment:
Instead of all this, why can't we just add a `onMouseDown` on the `popupIndicator` that would stop propagating?

Reply:
Probably for the `cancelIndicator` too